dns/py/params_test.py

```diff
@@ -73,6 +109,32 @@ def test_params(self):
         '-l,120,-Q,400,-d,/queries/abc',
         ','.join(inputs.dnsperf_cmdline))
 
+  def test_coredns_params(self):
+      values = {
+          'coredns_cpu': 100,
+          'coredns_cache': 200,
+      }
+
+      inputs = Inputs(make_mock_coredns_deployment_yaml(),
+          make_mock_coredns_configmap_yaml(), [])
```

Comment:
https://www.python.org/dev/peps/pep-0008/#indentation
Continuation lines should align wrapped elements either vertically using Python's implicit line joining inside parentheses, brackets and braces, or using a hanging indent.